windows/src/desktop/kmshell/xml/splash.xsl

```diff
@@ -30,7 +30,13 @@
   <div id="family"></div>
 
   <a href="keyman:start" id="startNow" class="button">
-    <div class="btn btn-blue"><xsl:value-of select="$locale/string[@name='S_Splash_Start']"/></div>
+    <div class="btn btn-blue">
+      <xsl:variable name="originalText" select="$locale/string[@name='S_Splash_Start_2']"/>
+      <xsl:variable name="replacement" select="'Keyman'"/>
+      <xsl:variable name="beforePlaceholder" select="substring-before($originalText, '%0:s')"/>
+      <xsl:variable name="afterPlaceholder" select="substring-after($originalText, '%0:s')"/>

```

Comment:
I am not sure what rule the XSL parser is following but the place holder is being modified so that it `Start %1$s` becomes `Start %0:s`

Reply:
Ok so this explains why it has been converted, therefore I it is deterministic why it this occurs. Add the update should work fine
https://github.com/keymanapp/keyman/blob/331c9d2d71bd209d7e85d1d11e6c742b85bffd89/common/windows/delphi/general/Keyman.System.AndroidStringToKeymanLocaleString.pas